libraries/RC_Channel/RC_Channel.h

```diff
@@ -173,6 +173,8 @@ class RC_Channel {
         SAILBOAT_TACK =       63, // rover sailboat tack
         REVERSE_THROTTLE =    64, // reverse throttle input
         GPS_DISABLE  =        65, // disable GPS for testing
+        RELAY5 =              66, // Relay5 pin on/off (in Mission planner set RC11_OPTION  = 66)
+        RELAY6 =              67, // Relay6 pin on/off (in Mission planner set RC12_OPTION  = 67)
```

Comment:
Please drop the Mission Planner part of the comment.

More importantly though, you are missing using this in RC_Channel.cpp, so it actually doesn't do anything.

Reply:
I've removed the mission planner text